docs/self-hosting/guides/monitoring-telemetry.mdx

```diff
@@ -37,287 +39,282 @@ OTEL_TELEMETRY_COLLECTION_ENABLED=true
+
+    3. **Create Prometheus configuration** for the collector:
+
+       ```yaml
+       global:
+         scrape_interval: 30s
+         evaluation_interval: 30s
+
+       scrape_configs:
+         - job_name: "otel-collector"
+           scrape_interval: 30s
+           static_configs:
+             - targets: ["otel-collector:8889"] # Adjust hostname/port based on your deployment
+           metrics_path: "/metrics"
+       ```
+
+       **Note**: Replace `otel-collector:8889` with the actual hostname and port where your OpenTelemetry Collector is running. This could be:
+
+       - **Docker Compose**: `otel-collector:8889` (service name)
+       - **Kubernetes**: `otel-collector.default.svc.cluster.local:8889` (service name)
+       - **Bare Metal**: `192.168.1.100:8889` (actual IP address)
+       - **Cloud**: `your-collector.example.com:8889` (domain name)
+
+    ### Deployment Options
+
+    <Tabs>
+      <Tab title="Docker Compose">
+        ```yaml
+        services:
+          otel-collector:
+            image: otel/opentelemetry-collector-contrib:latest
+            ports:
+              - 4318:4318 # OTLP http receiver
+              - 8889:8889 # Prometheus exporter metrics
+            volumes:
+              - ./otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml:ro
+            command:
+              - "--config=/etc/otelcol-contrib/config.yaml"
+        ```
+      </Tab>
+      <Tab title="Kubernetes">
+        ```yaml
+        # otel-collector-deployment.yaml
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: otel-collector
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: otel-collector
+          template:
+            metadata:
+              labels:
+                app: otel-collector
+            spec:
+              containers:
+                - name: otel-collector
+                  image: otel/opentelemetry-collector-contrib:latest
+                  ports:
+                    - containerPort: 4318
+                    - containerPort: 8889
+                  volumeMounts:
+                    - name: config
+                      mountPath: /etc/otelcol-contrib
+              volumes:
+                - name: config
+                  configMap:
+                    name: otel-collector-config
+        ```
+      </Tab>
+      <Tab title="Helm">
+        ```bash
+        helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+        helm install otel-collector open-telemetry/opentelemetry-collector \
+          --set config.receivers.otlp.protocols.http.endpoint=0.0.0.0:4318 \
+          --set config.exporters.prometheus.endpoint=0.0.0.0:8889
+        ```
+      </Tab>
+    </Tabs>
+
+  </Tab>
+</Tabs>
 
 ## Available Metrics
```

Comment:
we should break the metrics into two categories: system metrics and business metrics. what do you think?